### PR TITLE
v2.3.0-beta.2

### PR DESCRIPTION
**Pull request by S2Ler**

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 21.0.1
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 21.1.0-rc.1
```
Comment (MaximAlien) [inline Cartfile:1]: IIRC we still pin to exact versions of non-GA releases of our dependencies.

File: MapboxNavigation.podspec [hunk 2]
```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxMaps", "10.3.0-beta.1"
+  s.dependency "MapboxMaps", "10.3.0-rc.1"
```
Comment (bamx23) [inline MapboxNavigation.podspec:47]: We should either use `~>` here or even `>=` and `<` to match `from:` in SPM or use `exact` in SPM.

File: Package.swift [hunk 3]
```diff
@@ -22,10 +22,10 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.2.0"),
+        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.3.0-beta.2"),
```
Comment (bamx23) [inline Package.swift:25]: `exact`? As it's exact in Cathage and SPM.

**Review by bamx23 — APPROVED** [review]

File: MapboxNavigation.podspec (hunk 2 above)
Comment (S2Ler) [reply MapboxNavigation.podspec:47]: let's go with exact in spm

File: Package.swift (hunk 3 above)
Comment (S2Ler) [reply Package.swift]: makes sense, let's do it

**Comment by bamx23** [comment]
LGTM _(I've also tested for compatibility with the latest Search SDK, everything works fine)_

**Pull request merged**